packages/account/src/Types/common-prop.types.ts

```diff
@@ -0,0 +1,43 @@
+/** Add types that are shared between components */
```

Comment:
not exactly understand what is happening here :)
Recently you had created in this directory files common-prop.type.ts, where TFormValidation and TToken types were defined, and context.type.ts, where TPlatformContext and TApiContext were defined. 
what is the reason of creating new files and new types? could you olease explain  :)

Reply:
There is a separate PR with all configs required. But until it is merged, I need them here. Will ensure that PR gets merged first